src/components/CreatePostPrompt/CreatePostPrompt.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { Card, Form, Button } from 'react-bootstrap';
+import axios from 'axios';
+import { useCookies } from 'react-cookie';
+import { updateUserPosts } from '../../redux/userSlice';
+const CreatePostPrompt = ({ setNewPost }) => {
+    const [cookies] = useCookies(['authToken']);
+    const TOKEN = cookies.authToken;
+    const [content, setContent] = useState('');
+    const [image, setImage] = useState('');
+    const dispatch = useDispatch();
+    function createPostHandler(e) {
+        e.preventDefault();
+        const postData = new FormData();
+        if (content) postData.append('content', content);
+        else {
+            return;
+        }
```

Comment:
if else braces/brackets are not required for a single line of code.

Reply:
fixed